m4_sessions/1710-inning/group_four.md

```diff
@@ -1,31 +1,80 @@
-****Group Four****
-
-**Members**
+**Group Four Members**
 - David Becker
 - Jon Beckman
 - Sam Johnson
-- Yohanan Assefa
 - Michael Sagopolutele
 - Daniel Vogelsang
 
-Session Title: Our new role in social movements developers
-Description: Why should students come to this session?
+**Session Title: Social Movements And Technology**
+
+**Description:**
+
+In an increasingly technological world, there is an opportunity for those of us with the ability to write in the language of technology to play an oversized role in the direction of humanity. In this session, participants will be exposed to a number of social movements and the ways technology has played a role in them and then be given the opportunity to explore their own interests and ways they can make an impact. Participants will end the session with a tangible plan of action for their own personal involvement as developers after Turing.
+
+**Session Outcomes:** What will students leave with or gain from this session?
+
+A personal mission statmement/committment to the role they will play as a developer in making their communities a better place
+
+**Pre-Work/Communication**
+- Ask people to sign-up for [Code For Social Good](https://app.code4socialgood.org/) 
+- Ask for people to poke around on the site and pick out a project they are interested in
+- We will also include the other links from below for people to review as option 
+
+**Deliverables**
+- Pre-session message w/ link to Code For Social Good (Daniel)
+- Worksheet for "Active" discussion (Sam)
+
+**Session Outline:**
+
+**1. Intro** (5 minutes)
+
+Quick intros of presenters, participants, and the topic for the hour
+
+**2. Inform:** (20 minutes)
+
+Examples of social movements and the role of technology 
+
+_Present for ~5 minutes on each of the below social movements and how it helps inform our commitments to the greater good of our communities_
```

Comment:
This is excellent.  I love that you have broken this out so much and that you are providing multiple examples.